### ISSUE TITLE
please rephrase website

### ISSUE THREAD

**Issue by pryrt** [notepad-plus-plus/notepad-plus-plus#12026]
@donho,

Currently, the [online help](https://notepad-plus-plus.org/online-help/) page of the website says, 
> **[Community](https://community.notepad-plus-plus.org/)**
> Where you can get the support and find the answers of your questions.

Screenshot:
![image](https://user-images.githubusercontent.com/17455758/186024418-e7e57159-5d76-4598-acfa-4e9f3d3d3192.png)

Users are then understandably confused when the Community members tell them that it's not "official support" in the way they are thinking -- in my experience, in most users' minds, "support" means that they can report issues in the Community and expect them to be fixed in the application.  By not using the word "support" when describing the Community, it will manage user expectations.  ([Example here](https://community.notepad-plus-plus.org/post/79265))

So could you rephrase the description of the Community to be
> **[Community](https://community.notepad-plus-plus.org/)**
> Where you can ask fellow users for help and suggestions.

I would also suggest a new heading and description on that page
> **[Issue Tracker](https://github.com/notepad-plus-plus/notepad-plus-plus/issues/)**
> Where you can search through and make official feature requests and bug reports.

**Comment by alankilborn** [comment]
I might go even further to make it stand out more:

* Community: Where you can ask **FELLOW Notepad++ USERS** for help and suggestions.

* Issue Tracker: Where you can search through and make official feature requests and bug reports to the **Notepad++ DEVELOPERS**.

**Comment by donho** [comment]
@pryrt 
Makes sense Peter. Updated now.
